apps/testingaccessibility/src/pages/learn/[module]/[section]/[lesson]/index.tsx

```diff
@@ -72,20 +57,8 @@ const sectionQuery = groq`*[_type == "section" && slug.current == $slug][0]{
   }`
 
 export const getServerSideProps: GetServerSideProps = async ({req, params}) => {
-  const sessionToken = await getDecodedToken(req)
-  const {getPurchasesForUser} = getSdk()
-  const purchases =
-    sessionToken &&
-    sessionToken.sub &&
-    (await getPurchasesForUser(sessionToken.sub))
-  const productId = purchases && get(last(purchases), 'productId')
-
-  // fetch product from sanity based on user's productId associated with their purchase
-  const product = await sanityClient.fetch(productQuery, {
-    productId: productId,
-  })
-
   // get array of available lessons
+  const {product} = await getPurchasedProduct(req)
```

Comment:
this pattern was repeated in several places so I consolidated it. This should also make it simpler to add more permission stuff.